geny/src/geny/Writable.scala

```diff
@@ -134,5 +134,6 @@ object Readable{
 
   implicit class InputStreamReadable(i: InputStream) extends Readable{
     def readBytesThrough[T](f: InputStream => T): T = f(i)
+    override def httpContentType = Some("application/octet-stream")
```

Comment:
This breaks binary compatibility since it changes the return type from `Option[String]` to `Some[String]`.
Using explicit return type makes MiMa happy:
```suggestion
    override def httpContentType: Option[String] = Some("application/octet-stream")
```